ibm_catalog.json

```diff
@@ -284,6 +284,13 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
```

Comment:
I would not expose this input directly at the top level of the stack as it is unlikely to be common to change it.